### DISCUSSION TITLE
Not Received The GitHub Student Benefits

### DISCUSSION THREAD

**Discussion by ghost** [community/community#141913] [GitHub Education]
### Hi everyone,

 I apply for the GitHub student developer pack and I got approved as well but I have not received the benefits yet. Please help me to get the benefits so I can start learning.

**Comment by davevad93** [comment]
Hi @itsfatimarahman04 ,

When the benefits become available, the status label will turn purple and change to:

- `Congrats! Your request for @itsfatimarahman04 was approved and your benefits are now available.`

The waiting time may vary depending on different factors, such as the volume of requests; it usually takes a few days, but sometimes it can take a little longer, even a couple of weeks.

For further reference you can look at these discussions: #132790 , #131665 , #132470 , #133531, #134279, #135849, #136887, #138721, #139268

And also the [Student Developer Pack Application FAQs](https://github.com/orgs/community/discussions/111352) :

- [How long after I submit my application will it be approved?](https://github.com/orgs/community/discussions/111352#how-long-after-i-submit-my-application-will-it-be-approved)
- [How long after I’ve been approved will I receive my academic benefits?](https://github.com/orgs/community/discussions/111352#how-long-after-ive-been-approved-will-i-receive-my-academic-benefits)

**Reply by ghost** [reply]
Thank You So much I dear for your help . I will wait

**Reply by davevad93** [reply]
You're welcome @itsfatimarahman04 👍
